Add note to submit event before 1st (#2008)
betterscientificsoftware/bssw.io: docs/pages/bssw/bssw_eventworkflow.md

  * If so, construct the event page. Verify that the event is in .md format and properly formatted, as per the event style guide: https://betterscientificsoftware.github.io/bssw.io/bssw_styling_event.html 
  * Create a pull request

After the pull request is created, the editorial team will review and publish the event.

> NOTE: Event advertisment articles must be submitted by the 20th of the month before the event date so that they will be highlighted in the BSSw.io Digest email to the BSSw.io subscribers email list that goes out at the end of every month.  (Otherwise, event advertisment articles posted to BSSw.io may have much less visibility and reach a much smaller audience.  The only people who will see the event advertisement ) 

Reviewer: ```suggestion
NOTE: For maximal effect, event announcements should be submitted by the 20th of the month _before_ the event date.
Then, they are to be highlighted in the BSSw.io Digest email to the BSSw.io subscribers email list that goes out at the end of every month.
Otherwise, event announcements may have less visibility and reach a smaller audience. 
```

Author: I accepted this change in https://github.com/betterscientificsoftware/bssw.io/pull/2008/commits/313ffa1adc2c3a654e69a4a716047b8917ea7e30.  Thanks @markcmiller86!
